csharp/device/Microsoft.Azure.Devices.Client/IotHubConnectionString.cs

```diff
@@ -35,11 +43,16 @@ public IotHubConnectionString(IotHubConnectionStringBuilder builder)
             this.DeviceId = builder.DeviceId;
 #if WINDOWS_UWP
             this.HttpsEndpoint = new UriBuilder("https", builder.HostName).Uri;
-#else
+#elif !NETMF
             this.HttpsEndpoint = new UriBuilder(Uri.UriSchemeHttps, builder.HostName).Uri;
+#elif NETMF
+            this.HttpsEndpoint = new Uri("https://" + builder.HostName);
 #endif
-#if !WINDOWS_UWP
+
+#if !WINDOWS_UWP && !NETMF
             this.AmqpEndpoint = new UriBuilder(CommonConstants.AmqpsScheme, builder.HostName, AmqpConstants.DefaultSecurePort).Uri;
+#elif NETMF
+            //this.AmqpEndpoint = "amqps://" + this.HostName + ":" + AmqpConstants.DefaultSecurePort;
```

Comment:
I'd rather remove it and it can be added at the same time as AMQP. leaving commented code is confusing because it doesn't say anything about the state of the SDK.